src/vc_util/src/lib.rs

```diff
@@ -217,25 +226,12 @@ pub fn verify_ii_presentation_jwt_with_canister_ids(
     )?;
     let alias_tuple = get_verified_id_alias_from_jws(
         id_alias_vc_jws.as_str(),
+        &effective_vc_subject,
         &vc_flow_signers.ii_canister_id,
         root_pk_raw,
         current_time_ns,
     )
     .map_err(PresentationVerificationError::InvalidIdAliasCredential)?;
-    // TODO: change `get_verified_id_alias_from_jws` to additionally take `effective_vc_subject`
-    //       as an argument and to perform the check below internally.
-    //       Also, consider adding `ii_origin`-argument to enable custom values for II-origin,
```

Comment:
> Also, consider adding `ii_origin`-argument

Are we no longer considering that?

Reply:
I did consider this 😉 , and decided not to do it, the reason being that it is not needed right now, and it is not sure it will be needed. Also, the function has already 5 arguments, so it is getting "crowded".